lib/cybersource/ext/active_merchant/active_merchant.rb

```diff
@@ -70,6 +71,12 @@ def add_network_tokenization(xml, payment_method, options)
       end
 
       # Changes:
+      #  * http://apps.cybersource.com/library/documentation/dev_guides/Android_Pay_SO_API/html/wwhelp/wwhimpl/js/html/wwhelp.htm#href=ch_soAPI.html
+      #  * add paymentSolution tag to support Android Pay
+      def add_payment_solution(xml, payment_method)
+        xml.tag!("paymentSolution", "006") if (!payment_method.source.nil? && payment_method.source == :android_pay)
```

Comment:
Based on the code [cyber_source.rb](https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateways/cyber_source.rb#L507)